FirebaseSharedSwift/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 9.0.0
+- [added] **Breaking change:** `FirebaseSharedSwift` has exited beta.
```

Comment:
Did we actually make any breaking API changes?

Reply:
Hmm no. I thought the "breaking change" would be due to removing the `-beta` suffix from the tag but it doesn't look like we ever had a `-beta` suffix for this library.

Reply:
```suggestion
- [added] `FirebaseSharedSwift` has exited beta.
```